src/templates/components/container.html

```diff
@@ -0,0 +1,12 @@
+<div class="panel panel-default" ng-controller="formioContainerComponent">
```

Comment:
No problem Travis. I just used panel for UI enhancement. I will change it to dotted border based on your suggestion.